crates/bevy_pbr/src/ssao/mod.rs

```diff
@@ -460,22 +515,39 @@ impl SpecializedComputePipeline for SsaoPipelines {
                 "SAMPLES_PER_SLICE_SIDE".to_string(),
                 samples_per_slice_side as i32,
             ),
+            ShaderDefVal::UInt(
+                "METHOD".to_string(),
+                match key.ssao_settings.method {
+                    ScreenSpaceAmbientOcclusionMethod::Gtao => 0,
+                    ScreenSpaceAmbientOcclusionMethod::Vbao { .. } => 1,
+                },
+            ),
+            ShaderDefVal::UInt(
```

Comment:
I do not feel like this should be a shaderdef. I feel like we should probably use a uniform buffer / push constant here. Push constant is probably easier, and SSAO is not supported on WebGPU atm anyways.